schedule/yast/btrfs/btrfs+warnings_opensuse.yaml

```diff
@@ -46,10 +49,25 @@ test_data:
     - name: vda
       partitions:
         <<: !include test_data/yast/btrfs/common/btrfs+warnings_partitions.yaml
+        rootfs_small:
+          - role: raw-volume
+            size: 2mb
+            id: bios-boot
+          - role: operating-system
+            size: 4GiB
+            formatting_options:
+              should_format: 1
+              filesystem: xfs
+            mounting_options:
+              should_mount: 1
+              mount_point: /
   errors:
     <<: !include test_data/yast/btrfs/common/btrfs+warnings_errors.yaml
   warnings:
     <<: !include test_data/yast/btrfs/common/btrfs+warnings_warnings.yaml
     no_boot: A partition of type BIOS Boot Partition is needed to install the bootloader
     bios_boot_small_for_bootloader: A partition of type BIOS Boot Partition is needed to install the bootloader
     uefi_small: A partition of type BIOS Boot Partition is needed to install the bootloader
```

Comment:
if these three are always the same, why not just have one key?

Reply:
Good point! I've replaced all these keys with the one key. I'll re-run the VRs.